### ISSUE TITLE
Guarantee full execution of ban

### ISSUE THREAD

**Issue by TimDaub** [oceanprotocol/list-purgatory#39]
In #23, an account owner was banned but not their data sets. This issue was addressed months later in #37. I've not been part of the situation that lead to the account owner's ban, but I was naturally assuming that when an owner's account is banned, so are all their data sets.

I'm aware that there may be exceptions to this specific case, but using this repository's data, I'd be confident knowing that an account ban always includes a data set ban too. I'm arguing for this, as rugpullindex.com showed the banned owner's data sets for a few days without anyone realizing that they belong to a banned account. Rugpullindex.com has always filtered for both DIDs and accounts from this repository.

I think that a GitHub Action that runs the proposed account through the marketplace data to verify all data sets of an account have been banned would be an appropriate and automated solution we can all rely upon. Please let me know if that's of interest and I can think of sending a PR.

**Comment by trentmc** [comment]
> I was naturally assuming that when an owner's account is banned, so are all their data sets.

Yes, this should be the case, according to the rules laid out in this repo's README.

> I think that a GitHub Action that runs the proposed account through the marketplace data to verify all data sets of an account have been banned would be an appropriate and automated solution we can all rely upon.

I like this idea. 

Thoughts @kremalicious @brucepon ?

**Comment by TimDaub** [comment]
Getting a list of assets for an owner:

- https://market.oceanprotocol.com/search/?owner=addr

And the other way around, getting the owner for an asset should be straight-forward given that their address likely in the DDO somewhere.
